ocs_ci/ocs/platform_nodes.py

```diff
@@ -2225,3 +2226,23 @@ def stop_nodes(self, nodes, timeout=600, wait=True, force=True):
             wait_for_nodes_status(
                 node_names=node_names, status=constants.NODE_NOT_READY, timeout=timeout
             )
+
+    def restart_nodes_by_stop_and_start_teardown(self):
+        """
+        Make sure all RHV VMs are up by the end of the test
+
+        """
+        vms = self.rhv.get_vm_names()
+        assert vms, f"Failed to get VM list"
+
+        stopped_vms = [
+            vm
+            for vm in vms
+            if self.rhv.get_vm_status(vm) == VmStatus.DOWN
+            or self.rhv.get_vm_status(vm) == VmStatus.POWERING_DOWN
+        ]
+        # Start the VMs
+        if stopped_vms:
+            logger.info(f"The following VMs are powered off: {stopped_vms}")
+            self.rhv.start_rhv_vms(stopped_vms)
```

Comment:
I think that if a VM is in POWERING_DOWN status, it would not be possible to start it. 
If that's true, maybe add a wait mechanism for those POWERING_DOWN to reach DOWN state, and then start them

Reply:
incorporated review comments